### PR TITLE
Refactor gradle configuration

### PR DESCRIPTION
**Pull request by antban**
Refactor gradle configuration

### CODE REVIEW

File: api-metastore/src/main/java/org/zalando/nakadi/service/SubscriptionTimeLagService.java [hunk 1]
```diff
@@ -75,6 +75,9 @@ public SubscriptionTimeLagService(final TimelineService timelineService,
                     .allOf(futureTimeLags.values().toArray(new CompletableFuture[futureTimeLags.size()]))
                     .get(timeLagHandler.getRemainingTimeoutMs(), TimeUnit.MILLISECONDS);
 
+//            for (final Map.Entry<EventTypePartition, CompletableFuture<Duration>> entry: futureTimeLags.entrySet()) {
```
Comment (adyach) [inline api-metastore/src/main/java/org/zalando/nakadi/service/SubscriptionTimeLagService.java]: niiiiice

**Comment by adyach** [comment]
maybe you could move dependencies to separate file in the future

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**